plugins/fishlim/plugin_hexchat.c

```diff
@@ -138,7 +138,7 @@ static int handle_outgoing(char *word[], char *word_eol[], void *userdata) {
 static int handle_incoming(char *word[], char *word_eol[], hexchat_event_attrs *attrs, void *userdata) {
     const char *prefix;
     const char *command;
-    const char *recipient;
+    const char *channel = hexchat_get_info(ph, "channel");
```

Comment:
Ya I just used what handle_outgoing was using to get channel name and it worked, I didn't have any problems. There must be a better way to get channel name?